modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase

```diff
@@ -1511,6 +1511,67 @@ definition {
 
 	}
 
+	@description = "LPS-98517 - Verify that the long usernames in the result table do not break into different lines"
+	@priority = "3"
+	test LongUserNameDisplaysEllipsis {
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Site Administration",
+			portlet = "Workflow");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		JSONUser.addUser(
+			userEmailAddress = "buyerbuyer@liferay.com",
+			userFirstName = "buyerbuyerbuyerbuyerbuyer",
+			userLastName = "buyerbuyerbuyerbuyerbuyer",
+			userScreenName = "buyerbuyerbuyerbuyerbuyer");
+
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry Content",
+			entryTitle = "Blogs Entry Title");
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Users",
+			panel = "Control Panel",
+			portlet = "Users and Organizations");
+
+		User.assignRegularRoleCP(
+			roleTitle = "Administrator",
+			userScreenName = "buyerbuyerbuyerbuyerbuyer");
+
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
+
+		Workflow.assignTaskToUser(
+			assetTitle = "Blogs Entry Title",
+			assetType = "Blogs Entry",
+			assigneeName = "buyerbuyerbuyerbuyerbuyer (buyerbuyerbuyerbuyerbuyer buyerbuyerbuyerbuyerbuyer)",
+			workflowTask = "Review");
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Metrics");
+
+		WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
+
+		Click(
+			key_assigneeName = "Total Pending",
+			locator1 = "WorkflowMetrics#METRICS_WORKLOAD_BY_ASSIGNEE_NAME_TABLE");
+
+		AssertCssValue(
+			key_stepName = "buyerbuyerbuyerbuyerbuyer",
+			locator1 = "WorkflowMetricsPerformanceByStep#METRICS_PERFORMANCE_BY_STEP_PAGE_TABLE_STEP_NAME",
+			locator2 = "max-width",
+			value1 = "106.887px");
+	}
```

Comment:
You have inserted your testcase on the LPS-98517 stub - LongUserNameIsDisplayedInMoreThanOneLineInTable, it should on LRQA-67728 - LongTextsDisplayEllipsis, and please alter the description too, to be related to the correct ticket.